.github/workflows/test_itensortdvp.yml

```diff
@@ -0,0 +1,47 @@
+name: Run ITensorTDVP tests (Unbuntu)
+
+on:
+  push:
+    branches:
+      - main
+    tags: '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ matrix.threads }} thread(s)
+    runs-on: ${{ matrix.os }}
+    env:
+      JULIA_NUM_THREADS: ${{ matrix.threads }}
+    strategy:
+      matrix:
+        version:
+          - '1.6'
+          - '1'
+        os:
+          - ubuntu-latest
+        threads:
+          - '2'
+        arch:
+          - x64
+        exclude:
+          # MacOS not available on x86
+          - {os: 'macOS-latest', arch: 'x86'}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - name: Install Julia dependencies and run tests
+        shell: julia {0}
+        run: |
+          using Pkg;
+          Pkg.activate(temp=true);
+          Pkg.develop(path="./NDTensors");
+          Pkg.develop(path=".");
+          Pkg.add("ITensorTDVP");
+          Pkg.test("ITensorTDVP");
+      - uses: julia-actions/julia-uploadcodecov@latest
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I think we should remove this part related to code coverage, it seems to be causing issues in the CI run.